ansible_image_validation/validate-vm-images.yaml

```diff
@@ -6,11 +6,12 @@
 ## and runs validations against them
 ---
 - hosts: all
-  gather_facts: False
+  gather_facts: True
```

Comment:
Do we need to change this value to 'True'?

Reply:
yes, actually this is needed to extract ansible_distribution_major_version